styles/style.css

```diff
@@ -118,6 +122,65 @@ header .desktop-menu-container {
   display: none;
 }
 
+.menu-warp {
+  width: 100vw;
+  height: 100vh;
+}
+
+.blurry {
+  filter: blur(6px);
+  -webkit-filter: blur(6px);
+}
+
+.mobile-menu-container {
+  position: absolute;
+  top: 0;
+  background: rgba(96, 112, 2550, 0.8);
+  display: none;
+  width: 100%;
+  height: 100%;
+  z-index: 10;
+  backdrop-filter: blur(0.5rem);
+}
+
+.close-button {
+  padding: 0.5rem;
+  color: var(--white-color);
+  background: none;
+  border: none;
+  font-size: 3rem;
+  cursor: pointer;
+  outline: inherit;
+  display: flex;
+  align-content: flex-end;
+  margin-right: 1rem;
+  margin-top: 3rem;
+  margin-left: 8rem;
+}
```

Comment:
- [ ] You have done a good job with your mobile menu :muscle: but there is a styling issue because the (X) icon to close the menu looks almost in the middle of the page and according to the Figma design it should be aligned to the right (considering the same spacing that is being used to the left side). Please, kindly fix this style issue to match the template :wink: :raised_hands: 

| Your design | Figma design |
| --- | --- |
| ![image](https://user-images.githubusercontent.com/81522435/146044539-d70f5708-3288-455a-8d5b-75563e7bbe31.png) | ![image](https://user-images.githubusercontent.com/81522435/146044376-2015496a-a50a-4783-82df-e9b0c87627a3.png) |

Reply:
Hi @alonsofl, Thanks for your feedback, I will check that!